docs/cookbook/docker_logs.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Sending Docker Logs to Axiom
+---
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+
+<head>
+  <title>Sending Docker logs to Axiom</title>
+</head>
+<p>
+
+To set up sending Docker logs to Axiom using the provided Docker Compose file, daemon.json, and logstash.config, follow these step-by-step instructions:
```

Comment:
Can you add a small note for docker compose version as well. Older docker-compose does not show logs. 1.29 works fine.

Reply:
Can you also provide a link to docs on axiom, where it specifies how to create a token and database?

Reply:
Also, add in a tip: "To use axiom's free tier, it's better to create a new email and share between users"